.github/workflows/docs.yml

```
@@ -4,9 +4,20 @@ on:
   push:
     branches:
       - main
-    tags: '*'
+    tags: "*"
   pull_request:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# We need spack so that we can install palace for the tutorials
+env:
+  SPACK_VERSION: develop
+  REGISTRY: ghcr.io/awslabs/palace
+  GITHUB_USER: ${{ github.actor }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
 jobs:
   build-docs:
     permissions:
```

```suggestion
    permissions:
      packages: write
```

This is the only other thing that I can think of from the slack thread... It seems like we are doing everything right.

The other alternatives I can think of are:
- Just to abandon GHCR for this and use an S3 bucket
- Use different GHCR caches for each workflow

I maintain that this should just work....